Documentation/ceph-csi-drivers.md

```diff
@@ -108,105 +121,109 @@ Once the plugin is successfully deployed, test it by running the following examp
 
 # Test RBD CSI Driver
 
-## Create RBD StorageClass
-
-This
-[storageclass](../cluster/examples/kubernetes/ceph/csi/example/rbd/storageclass.yaml)
-expects a pool named `rbd` in your Ceph cluster. You can create this pool using
-[rook pool
-CRD](https://github.com/rook/rook/blob/master/Documentation/ceph-pool-crd.md).
-
-Please update `monitors` to reflect the Ceph monitors.
+## Create a CephBlockPool for our storage class to provision PVs from
 
-```console
-kubectl create -f cluster/examples/kubernetes/ceph/csi/example/rbd/storegeclass.yaml
+```bash
+cat << EOF | kubectl create -f -
+apiVersion: ceph.rook.io/v1
+kind: CephBlockPool
+metadata:
+  name: rbd
+  namespace: rook-ceph
+spec:
+  failureDomain: host
+  replicated:
+    size: 3
+EOF
 ```
 
-## Create RBD Secret
+## Create a Secrets object (plugin will access our Ceph cluster via this secrets file)
 
-Create a Secret that matches `adminid` or `userid` specified in the
-[storageclass](../cluster/examples/kubernetes/ceph/csi/example/rbd/storageclass.yaml).
+We need credential for two users on the Ceph Cluster; *admin* and *kubernetes*.  We can obtain these from the Operator
+POD using the ceph toolbox. 
```

Comment:
```suggestion
pod using the ceph toolbox. 
```

Reply:
The sentence is a bit confusing, it's either from the operator or the Ceph toolbox.